package.json

```diff
@@ -52,10 +52,12 @@
     "@lezer/highlight": "1.1.6",
     "@lit-labs/context": "0.4.1",
     "@lit-labs/motion": "1.0.4",
+    "@lit-labs/observers": "2.0.1",
```

Comment:
FYI, this introduced a duplicate of `@lit/reactive-element` (bumping it to the next major version).

Reply:
Adjusted in #18484